templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -95,7 +96,7 @@ export default function ChatInput(
       )}
       {files.length > 0 && (
         <div className="flex gap-4 w-full overflow-auto py-2">
-          {files.map((file, index) => (
+          {files.map((file: DocumentFile) => (
             <DocumentPreview
               key={file.metadata?.id ?? `${file.filename}-${index}`}
```

Comment:
```suggestion
              key={file.id}
```

Reply:
we also have `key={file.metadata?.id ?? `${file.filename}-${index}`}` somewhere else - also change it